signac/contrib/project.py

```diff
@@ -1827,3 +1857,16 @@ def get_project(root=None):
     :rtype: :py:class:`~.Project`
     :raises LookupError: If no project configuration can be found."""
     return Project.get_project(root=root)
+
+
+def get_job(root=None):
+    """From a job workspace directory, locate the closest parent project
```

Comment:
I think this is supposed to express "when the current working directory is a workspace directory" or so, but I don't think this is clear with the current description.

Reply:
I changed this.